tokenizer/pom.xml

```diff
@@ -15,17 +15,17 @@
         <dependency>
             <groupId>edu.illinois.cs.cogcomp</groupId>
             <artifactId>illinois-core-utilities</artifactId>
-            <version>3.1.12</version>
+            <version>3.1.13</version>
         </dependency>
         <dependency>
             <groupId>edu.illinois.cs.cogcomp</groupId>
             <artifactId>LBJava-NLP-tools</artifactId>
-            <version>3.1.12</version>
```

Comment:
not sure why all these pom version changes show up; in general, we leave the version as is, and it gets incremented by a deploy build operation on semaphore/jenkins.  Please reset to whatever the current version is.